streams/src/main/java/org/apache/kafka/streams/kstream/internals/suppress/SuppressedInternal.java

```diff
@@ -62,22 +74,25 @@ public boolean equals(final Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         final SuppressedInternal<?> that = (SuppressedInternal<?>) o;
-        return Objects.equals(bufferConfig, that.bufferConfig) &&
+        return suppressTombstones == that.suppressTombstones &&
+            Objects.equals(getName(), that.getName()) &&
```

Comment:
why `getName()` instead of `name` ? Similar for others.

Reply:
ok, I've changed them.